provisioning/miniprov/hare_mp/main.py

```diff
@@ -588,6 +588,30 @@ def reset(args):
         raise RuntimeError(f'Error during reset : {error}')
 
 
+@func_log(func_enter, func_leave)
+@repeat_if_fails()
+def cleanup_node_facts(utils: Utils, cns_utils: ConsulUtil):
+    hostname = utils.get_local_hostname()
+    keys: List[KeyDelete] = [
+        KeyDelete(name=f'{hostname}/facts', recurse=True),
+    ]
+
+    if not cns_utils.kv.kv_delete_in_transaction(keys):
+        logging.error('Delete transaction failed for %s', keys)
+
+
+@func_log(func_enter, func_leave)
+@repeat_if_fails()
+def cleanup_node_disks(utils: Utils, cns_utils: ConsulUtil):
+    hostname = utils.get_local_hostname()
+    keys: List[KeyDelete] = [
+        KeyDelete(name=f'{hostname}/', recurse=True),
```

Comment:
deleting `{hostname}/'` recursively deletes `{hostname}/facts'` keys too. 

`cleanup_node_facts()` will be unnecessary and can be removed.

Reply:
As per discussion in DSU, Using two different functions for removing `{hostname}/facts` and `{hostname}/drives`. So it will be removed only particular data from consul kv.

Reply:
ok thanks